packages/eas-cli/src/build/ios/configure.ts

```diff
@@ -0,0 +1,73 @@
+import { CredentialsSource, Workflow } from '@eas/config';
+import { ExpoConfig, IOSConfig } from '@expo/config';
+import assert from 'assert';
+
+import { readRawAsync as readRawCredentialsJsonAsync } from '../../credentials/credentialsJson/read';
+import * as ProvisioningProfileUtils from '../../credentials/ios/utils/provisioningProfile';
+import { getProjectAccountNameAsync } from '../../project/projectUtils';
+import { confirmAsync } from '../../prompts';
+import { ConfigureContext } from '../context';
+import { isExpoUpdatesInstalled } from '../utils/updates';
+import { configureUpdatesAsync, syncUpdatesConfigurationAsync } from './UpdatesModule';
+import { getBundleIdentifier } from './bundleIdentifer';
+import { resolveIosCredentialsAsync } from './credentials';
+
+export async function configureIosAsync(ctx: ConfigureContext): Promise<void> {
+  if (!ctx.hasIosNativeProject) {
+    return;
+  }
+  const bundleIdentifier = await getBundleIdentifier(ctx.projectDir, ctx.exp);
+  IOSConfig.BundleIdenitifer.setBundleIdentifierForPbxproj(ctx.projectDir, bundleIdentifier, false);
+
+  await generateBestEffortSigningConfigAsync(ctx);
+
+  if (isExpoUpdatesInstalled(ctx.projectDir)) {
+    await configureUpdatesAsync(ctx.projectDir, ctx.exp);
+  }
+}
+
+export async function syncProjectConfigurationAsync(
+  projectDir: string,
+  exp: ExpoConfig
+): Promise<void> {
+  // TODO: check bundle identifier
+  if (isExpoUpdatesInstalled(projectDir)) {
+    await syncUpdatesConfigurationAsync(projectDir, exp);
+  }
+}
+
+async function generateBestEffortSigningConfigAsync(ctx: ConfigureContext): Promise<void> {
```

Comment:
Maybe just `configureXcodeProjectAsync`? 

I think the name you proposed is a little confusing mostly because this function is not **generating** anything.